src/components/LocalLabelAppBadge/LocalLabelPopoverActionLabel.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
```

Comment:
It'd be nice if the `LocalIdenittyBadge` also used these components; there's some subtle differences.